code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -775,7 +775,7 @@
 	description = "Causes an electrical surge in the targeted cyborg, rebooting and repairing most of its subsystems. Requires two uses on a cyborg with broken armor."
 	cost = 20
 	power_type = /datum/action/innate/ai/ranged/repair_cyborg
-	unlock_text = "<span class='notice'>TLB exception on load: Error pointing to address 0000001H, Proceed with execution anywa- SURGE protocalls installed, welcome to open APC!</span>"
```

Comment:
This was supposed to be a pun but I don't think it landed tbh

Reply:
Can you explain the pun?